aep/general/0132/aep.md.j2

```diff
@@ -1,22 +1,26 @@
-# GET for collections
+# List
 
 In REST APIs, it is customary to make a `GET` request to a resource
 collection's URI (for example, `/v1/publishers/{publisher}/books`) in order to
-retrieve a list of the resources within that collection. These are sometimes
-colloquially called "list" operations.
+retrieve a list of the resources within that collection.
+
+Resource-oriented design (AEP-121) honors this pattern through the `List`
+method. These RPCs accept the parent collection (and potentially some other
+parameters), and return a list of responses matching that input.
 
 ## Guidance
 
-APIs **should** generally provide a `GET` method for resource collections
-unless it is not valuable for users to do so. When the `GET` method is used on
-a URI ending in a resource collection, the result should be a list of
-resources. For more information about using the `GET` method on a URI ending
-with a discrete resource ID, see AEP-131.
+APIs **should** provide a `List` method for resource collections.The purpose of
+the `List` method is to return data from a finite collection (generally
+singular unless the operation supports [reading across collections][]).
+
+When the `GET` method is used on a URI ending in a resource collection, the
+result **must** be a list of resources.
 
 ### Requests
 
-Collection `GET` operations **must** be made by sending a `GET` request to the
-resource collection's URI:
+`List` operations **must** be made by sending a `GET` request to the resource
+collection's URI:
 
 ```http
 GET /v1/publishers/{publisher}/books HTTP/2
```

Comment:
nit: dding "HTTP/2" here feels weird to me. I know it was from the previous version though.

Reply:
I agree.  We need a broader style pass on the standard method AEPs to get them stylistically consistent.  Are you okay with doing that in a separate PR?

Reply:
I'd rather it just be removed now. But alternatively if you can file an issue it's fine.

Reply:
One nice thing is that if it's an issue, we could put it in the 'starter tasks' for the barn-raising.

Reply:
Created an issue and tagged accordingly, thanks.